projects/uitdatabank/docs/entry-api/places/booking-availability.md

```diff
@@ -0,0 +1,6 @@
+Coming soon.
+
+<!-- 
+  @todo 
+  Explain that it is currently not supported to mark a place as sold out, as the status is top-level and a place can only be sold out for a specific moment/day and this is currently not possible. 
```

Comment:
I thought it best to clarify this because places do have a `bookingAvailability` but it's always set to `Available` and some partners/integrators might wonder why they cannot change it?

Reply:
In the meantime I'm going to merge this PR but we can of course still continue this conversation and later delete the page if needed.